CMakeLists.txt

```diff
@@ -322,4 +328,5 @@ endif(ENABLE_INSTALLER)
 
 MESSAGE( STATUS "" )
 MESSAGE( STATUS "PROFILER: ${USE_PROFILER}" )
+MESSAGE( STATUS "query_txid: ${LOAD_QUERY_TXID_PLUGIN}")
```

Comment:
I think this should be:
```
MESSAGE( STATUS "LOAD_QUERY_TXID_PLUGIN: ${LOAD_QUERY_TXID_PLUGIN}")
```

So i can know exactly what variable i should change to turn it on. Rest looks all good, thanks.